grassroot-services/src/main/java/za/org/grassroot/services/livewire/LiveWireAlertBrokerImpl.java

```diff
@@ -530,6 +548,38 @@ public void reviewAlert(String userUid, String alertUid, List<String> tags, bool
         logsAndNotificationsBroker.asyncStoreBundle(bundle);
     }
 
+    @Override
+    public boolean isUserBlocked(String userUid){
+        User user = userRepository.findOneByUid(userUid);
+        boolean canCreate = true;
+
+        Optional<ConfigVariable> timesConfigVariable = configRepository.findOneByKey("times.livewire.user.blocked");
+        int timesUserBlocked = timesConfigVariable.map(var -> Integer.parseInt(var.getValue())).orElse(0);
```

Comment:
Don't make this default to zero, otherwise it will block everyone who tries anything (because the condition below will always be true).